docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+nfs:
+    image: misterbisson/nfsserver
+    privileged: true
+    mem_limit: 256m
+    restart: always
+    labels:
+      - triton.cns.services=nfs
+    expose:
+      - 111
+      - 1892
+      - 2049
+    environment:
+      - CONSUL=consul
```

Comment:
I added this as a task in https://github.com/autopilotpattern/nfsserver/pull/1#issuecomment-205107727